docs/rfc/rfc-1.adoc

```diff
@@ -184,22 +187,48 @@ reason), all they need to do is not submit their reveal and wait 30 days.
 
 [[sweeping]]
 ==== Sweeping
+
+Governable Parameters
+
+- `sweeping_refund_safety_time`: The amount of time prior to when a UTXO
+  becomes eligible for a refund where we will not include it in a sweeping
+  transaction.
+- `sweep_period`: The amount of time we wait between sweeps.
+- `sweep_max_btc`: The amount of summed unswept bitcoin deposits that will
+  trigger an early sweep.
+- `dust_threshold`: The minimum bitcoin deposit amount for the transaction to
+  be included in a sweep.
+
 The operators sign a transaction that unlocks all revealed deposits, combines
 them into a single UTXO with the existing UTXO, and relocks that transactions
 without a 30-day refund clause to same wallet. This has two main effects: it
 consolidates the UTXO set and it disables the refund.
 
-Caveat: We only include deposits in batches that have at least 8 remaining
-hours on their refund window. This prevents potential attacks or corner cases
-where we create a transaction with a valid, unspent input, but by the time we
-have signed that transaction, the depositor has already submitted a refund to
-the mining pool. Giving ourselves 8 hours of leeway stops this from happening.
-Once a deposit crosses that 8-hour threshold, the depositor should refund their
+Caveat: We only include deposits in batches that have at least
+`sweeping_refund_safety_time` their refund window. This prevents potential
+attacks or corner cases where we create a transaction with a valid, unspent
+input, but by the time we have signed that transaction, the depositor has
+already submitted a refund to the mining pool. Giving ourselves this leeway
+stops that from happening.  Once a deposit crosses that
+`sweeping_refund_safety_time` threshold, the depositor should refund their
 deposit.
```

Comment:
One more thing that I noticed - this sentence is a bit misleading. The depositor will not be able to process the refund directly after `sweeping_refund_safety_time` threshold is crossed. Threshold is crossed at `deposit_creation_time + 30 days - sweeping_refund_safety_time` and depositor can process a refund at `deposit_creation_time + 30 days`.

Reply:
addressed in 94e76fd